examples/explore_browser_use/initial_actions.py

```
@@ -0,0 +1,36 @@
+from browser_use.llm import ChatOpenAI
+from browser_use import Agent
+from dotenv import load_dotenv
+import os
+import sys
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+load_dotenv()
+
+import asyncio
```

Ruff (0.11.9)

Module level import not at top of file

(E402)